scripts/core/exp-command-buffer.yml

```diff
@@ -704,3 +704,36 @@ returns:
         - "If event objects in phEventWaitList are not valid events."
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get profiling information for the sync point execution associated with an event object"
+class: $xSyncPoint
+name: GetProfilingInfoExp
```

Comment:
I think it looks inconsistent that this `${x}SyncPointGetProfilingInfoExp` the only entry-point defined by the command-buffer extension that isn't prefixed with "CommandBuffer", I'd probably have a preference for this being called `${x}CommandBufferGetSyncPointProfilingInfoExp`.

We can ask the UR team before making this change because I realize it involves a lot of code churn.

Reply:
> We can ask the UR team before making this change because I realize it involves a lot of code churn.

Using the `$xEvent` class may be the way forward so that the correct function pointer table is used in the loader.

Reply:
The function is now part of the `$xEvent` class and its name has been updated accordingly. However, this function is not linked through the `Event` table but through a new table called `EventExp`. (see https://github.com/Bensuo/unified-runtime/pull/5/files#diff-6e641e57d1187e38f4b6022173a637db39a8fb38f0494a81cb627a2de3edbc86R283)